compiler/ilgen/MethodBuilder.hpp

```diff
@@ -73,7 +74,7 @@ class MethodBuilder : public TR::IlBuilder
 
    TR::TypeDictionary *typeDictionary()                      { return _types; }
 
-   const char *getDefiningFile()                             { return _definingFile; }
+   std::string getDefiningFile()                       { return _definingFile; }
```

Comment:
Can you fix the alignment here so that the `{` is aligned with the other ones (as it was before)?